docs/docsite/rst/collections_guide/collections_installing.rst

```diff
@@ -114,6 +169,32 @@ Downloading a collection for offline use
 
 .. include:: ../shared_snippets/download_tarball_collections.txt
 
+.. _collection_local_install:
+
+Installing collections adjacent to playbooks
+--------------------------------------------
+
+You can install collections locally next to your playbooks inside your project instead of in a global location on your system or on your AWX.
+This approach has these added benefits:
+* Ensures that all users of the project use the same collection version.
```

Comment:
A tip with list items is to use parallelism. The starting verb in each list item should use the same tense.